renderer/index.js

```diff
@@ -1019,6 +1025,12 @@ function openTorrentContextMenu (infoHash) {
   menu.popup(remote.getCurrentWindow())
 }
 
+function openDownloadFolder (torrentSummary) {
+  var folder = path.parse(torrentSummary.name).name
+  var fullPath = path.join(torrentSummary.path, folder)
```

Comment:
I've found that this approach doesn't work for all torrents. There must be a better way to find the path for the download location. Any pointers greatly appreciated 😃